files/build_templates/buffers_config.j2

```diff
@@ -188,7 +190,16 @@ def
     {%- if port_names_list_inactive.append(port) %}{%- endif %}
 {%- endfor %}
 {%- set port_names_inactive  = port_names_list_inactive  | join(',') %}
+
+{%- set port_names_list_all  = [] %}
```

Comment:
Shall we move code line 194-198 under the if branch {% if (defs.generate_cable_length_config is defined) %} at line 200?

Reply:
The macro invocation at line 201 will generate the config, if macro is defined, else the code at 192-199 will generate the CABLE_LENGTH per the original code.